terraform/modules/jenkins/flowlogs.tf

```diff
@@ -0,0 +1,54 @@
+resource "aws_flow_log" "jenkins_flowlog" {
+  iam_role_arn    = aws_iam_role.jenkins_flowlog_role.arn
+  log_destination = aws_cloudwatch_log_group.jenkins_flowlog_log_group.arn
+  traffic_type    = "ALL"
+  vpc_id          = aws_vpc.main.id
+}
+
+resource "aws_cloudwatch_log_group" "jenkins_flowlog_log_group" {
```

Comment:
Could you add the common tags to this resource? [Here's an example](https://github.com/nationalarchives/tdr-jenkins/blob/master/terraform/modules/jenkins/ecs.tf#L52), though I'm not sure why we're adding a `Name` tag to everything. @TomJKing - what do we use it for?

Ditto for the role below.

Reply:
Name is useful for some resources when you view them on the console. There is a particular resource, which I can't remember, where the tag name is used in the ui display. Happy to have it removed.